.eslintrc.js

```diff
@@ -29,13 +29,13 @@ module.exports = {
     '@typescript-eslint/no-base-to-string': 'warn',
     '@typescript-eslint/no-confusing-non-null-assertion': 'warn',
     '@typescript-eslint/no-dupe-class-members': 'error',
-    '@typescript-eslint/no-duplicate-imports': 'warn',
+    '@typescript-eslint/no-duplicates': 'warn',
     '@typescript-eslint/no-extra-parens': 'warn',
     '@typescript-eslint/no-floating-promises': 'off',
     '@typescript-eslint/no-inferrable-types': 'off',
     '@typescript-eslint/no-non-null-asserted-nullish-coalescing': 'warn',
     '@typescript-eslint/no-non-null-assertion': 'off',
-    '@typescript-eslint/no-parameter-properties': 'warn',
+    '@typescript-eslint/parameter-properties': 'error',
```

Comment:
Did you mean to change this from warn to error?

Reply:
good catch, fixed